functions-methods.js

```diff
@@ -33,4 +87,46 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+// checkEmailValidity("tessmellink@novinl") geeft false - want er staat geen . in
+
+console.log( "===========================  OPDRACHT 3 ============================" )
+
+
+
+function checkEmailValidity(email) {
+
+  const domain = email;
+  // console.log(domain);
+
+  // gestript email adres na @ teken ivm check op aanwezigheid .nl of .com of .info - niet op vaste positie ivm diverse domeinen. Als er een punt voor de @ wordt gebruikt wordt deze nu genegeerd.
+ 
+  const emailStripped = domain.substring(domain.indexOf("@") + 1);
+  // console.log(emailStripped);
+  let message = "";
+
+  if (domain.charAt(domain.indexOf("@")) !== "@" || 
+      domain.charAt(domain.indexOf(",")) === ",")  {
+    
+    message = "false";
+
+  } else if (domain.endsWith(".") ||
+             emailStripped.charAt(emailStripped.indexOf(".")) !== ".") {
+    
+    message = "false";
+    
+
+  }  else {
+
+    message = "true";
+
+  }
+
+  return message;
```

Comment:
Dit kun je nog wat versimpelen
```suggestion
  return domain.includes("@") && !domain.includes(",") && !domain.endsWith(".") && emailStripped.includes(".");
```